topics/ecology/tutorials/remote-sensing/tutorial.md

```diff
@@ -0,0 +1,437 @@
+   - Preprocess Sentinel 2 data
+ - **A second step** to compute biodiversity indicators
+   - Global overview
+   - For Canopy
+ - **A third step** to compute spectral indices:
+   - Spectral indices 
+   - EBV
+
+> <details-title>Details about spectral indices</details-title>
+>
+> Spectral indices are used to highlight particular features or properties of the earth's surface, e.g. vegetation, soil, water. They are developed on the basis of the spectral properties of the object of interest.
+>
+> Knowledge of the leaf cell, plant structure, state, condition and spectral properties is essential to perform vegetation analysis using remote sensing data.   
+> Spectral indices dedicated to vegetation analysis are developed on the basis that healthy vegetation reflects strongly in the near infrared (NIR) spectrum while absorbing strongly in the visible red.   
+>
+{: .details}
+
+
+In this tutorial, we'll be working on Sentinel 2 data extracted from the Copernicus portal, Scihub.  First those data will be prepared. After pre-processing to fit the input format of the tools, we'll see how to calculate biodiversity metrics.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Upload and pre-processing of the data
+
+This first step consist of downloading and properly prepare the data to use it in Sentinel 2 toolsuite.
+
+> <hands-on-title>Hands-on: Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “Sentinel 2 data for biodiversity tutorial”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md box_type="none" %}
+>
+>
+> 2. Download the files from [Scihub](https://scihub.copernicus.eu/dhus/#/home) or from [PEPS](https://peps.cnes.fr/rocket/#/search?maxRecords=50&page=1) :
+>
+>    You will have to to create an account for either of these platform. 
+>    Select Sentinel 2 and choose the Product type "S2MSI2A".
+>
+>    ![Scihub portal](../../images/remote_sensing/scihub.png "Scihub portal")
+>
+>    This an example of the Copernicus portal, Scihub. You need to download a zip folder. Keep it that way. 
+>
+> 3. Upload the zip folder
+>
+>     > <tip-title>Tip: Importing data from your computer</tip-title>
+>     >
+>     > * Open the Galaxy Upload Manager {% icon galaxy-upload %}
+>     > * Select **Choose local files**
+>     > * Browse in your computer and get the downloaded zip folder
+>     >
+>     > * Press **Start**
+>     {: .tip}
+>
+> 4. You can rename the dataset, “sentinel_2_data.zip" for example, and to keep informations about the original name "S2A_MSIL2A.....zip", you can copy/paste it on the "Info" box.
+>
+>
+{: .hands_on}
+
+# Preparing data
+
+Using preprocS2 R package, this step provides a unique tool to read, crop, resample the original image directory, and write it as a raster stack
+
+## **Preprocessing sentinel 2 data**
+
+> <hands-on-title>Hands-on: Preprocess</hands-on-title>
```

Comment:
```suggestion
> <hands-on-title>Preprocess</hands-on-title>
```

The `Hands-on:`, `Question:` etc part are already added automatically, so you can leave them out here and only add what you want to appear after that (or leave empty if nothing)